src/__tests__/sanity.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-ignore
```

Comment:
There is a problem with 'isolated-modules' [typescript docs](https://www.typescriptlang.org/v2/en/tsconfig#isolatedModules), which is a compiler option that is required with Next, it won't compile without it.

The problem is that files which are not imported anywhere, and do not import any other files will be identified as isolated modules. To this the ts compiler say "no, don't Since this test is just a sanity check it doesn't import anything, it makes things fail. And since we don't have any other test yet, I decided to keep it to have some boilerplate for when we might eventually need it?